lib/adiwg/mdtranslator/writers/mdJson/sections/mdJson_dataQualityReport.rb

```diff
@@ -17,12 +17,12 @@ module DataQualityReport
           def self.build(hReport)
             Jbuilder.new do |json|
               json.type hReport[:type]
+              json.qualityMeasure QualityMeasure.build(hReport[:qualityMeasure]) unless hReport[:qualityMeasure].nil?
               json.conformanceResult @Namespace.json_map(hReport[:conformanceResult], ConformanceResult) unless hReport[:conformanceResult].nil? || hReport[:conformanceResult].empty? 
+              json.coverageResult @Namespace.json_map(hReport[:coverageResult], CoverageResult) unless hReport[:coverageResult].nil? || hReport[:coverageResult].empty?
               json.descriptiveResult @Namespace.json_map(hReport[:descriptiveResult], DescriptiveResult) unless hReport[:descriptiveResult].nil? || hReport[:descriptiveResult].empty? 
-              json.evaluationMethod EvaluationMethod.build(hReport[:evaluationMethod]) unless hReport[:evaluationMethod].nil?
-              json.qualityMeasure QualityMeasure.build(hReport[:qualityMeasure]) unless hReport[:qualityMeasure].nil?
+              json.evaluationMethod @Namespace.json_map(hReport[:evaluationMethod], EvaluationMethod) unless hReport[:evaluationMethod].nil? || hReport[:evaluationMethod].empty? 
```

Comment:
Reorder these so they show up in the same order as listed in <insert issue link to data quality report>